api/app/Http/Controllers/UserController.php

```diff
@@ -2,15 +2,33 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\College;
+use App\Models\Course;
+use App\Models\Department;
 use App\Models\User;
+use Illuminate\Contracts\Database\Eloquent\Builder;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
 
 class UserController extends Controller
 {
-    //
+    public function getAuthUser()
+    {
+        $user = Auth::user();
+        $id = Auth::id();
+
+        if ($user->roleUser->role_id === 2) {
```

Comment:
create scope for this 
Ref: https://laravel.com/docs/9.x/eloquent#local-scopes
`$user->student`

Reply:
Comment unresolved. Unable to use scope to return truthy or falsy values.

Reply:
Comment resolved.